Tests/KituraTests/KituraTestBuilder.swift

```diff
@@ -84,6 +86,16 @@ class ServerTestBuilder: RequestTestBuilder, AssertionTestBuilder {
                 })
             }
         }
+        
+        init(_ test: KituraTest, _ method: String, _ path: String, URLEncodedObject: String) {
+            self.test = test
+            self.invoker = { callback in
+                test.performRequest(method, path: path, callback: callback, requestModifier: { request in
+                    request.headers["Content-Type"] = "application/x-www-form-urlencoded; charset=utf-8"
```

Comment:
According to https://stackoverflow.com/questions/16819502/application-x-www-form-urlencoded-and-charset-utf-8 the `application/x-www-form-urlencoded` MIME type doesn't have a `charset` parameter, so this should just be `request.headers["Content-Type"] = "application/x-www-form-urlencoded"` I think?